modules/tensor_mechanics/tests/1D_spherical/smallStrain_1DSphere.i

```diff
@@ -41,10 +41,9 @@
 []
 
 [Kernels]
-  [./StressDivergenceRSphericalTensors]
-    type = StressDivergenceRSphericalTensors
-    component = 0
-    variable = disp_r
+  [./RSpherical]
```

Comment:
Yeah, I liked the previous name better. It make more clear what physics the kernels actually represent.